client/src/components/Login.tsx

```diff
@@ -11,21 +14,33 @@ const Login = (): ReactElement => {
       justifyContent='center'
       minHeight='100vh'
     >
-      <Box>
-        <Typography variant='h2' align='center' gutterBottom>
-          <Link to='/'>
-            <h1>mkDev</h1>
-          </Link>
+      <Box alignContent={'center'}>
+        <Typography
+          variant='h4'
```

Comment:
For accessibility reasons, these should be h1s with their font sizes changed,

Reply:
Accessibility? So every text we implement needs to be an H1?